### ISSUE TITLE
allow_unknown in schema dict broken

### ISSUE THREAD

**Issue by lwcolton** [nicolaiarocci/cerberus#80]
The cerberus documentation gives an example of setting allow_unknown in your schema dictionary.  However, even when following the example verbatim, cerberus chokes

https://cerberus.readthedocs.org/en/latest/#allowing-the-unknown

```python
from cerberus import Validator
v = Validator()
schema = {
  'name': {'type': 'string'},
  'a_dict': {
    'type': 'dict',
    'allow_unknown': True,
    'schema': {
      'address': {'type': 'string'}
    }
  }
}
v.validate({'name': 'john', 'a_dict':{'an_unknown_field': 'is allowed'}}, schema)
```

produces

`cerberus.cerberus.SchemaError: unknown rule 'allow_unknown' for field 'a_dict'`

**Comment by nicolaiarocci** [comment]
Hello, on which Cerberus version are you?

**Comment by lwcolton** [comment]
$ pip show cerberus
---
Name: Cerberus
Version: 0.8.1

**Comment by nicolaiarocci** [comment]
This is fixed in 0.8.2-dev, see https://github.com/nicolaiarocci/cerberus/issues/75. Can you please try to pull from that and see if it works for you? I just tried with your example and it seems to be working fine. Thanks.
